sdk/translation/azure-ai-translation-text/tests/test_translation.py

```diff
@@ -290,7 +290,7 @@ def test_token(self, **kwargs):
         endpoint = kwargs.get("translation_text_endpoint")
         apikey = kwargs.get("translation_text_apikey")
         region = kwargs.get("translation_text_region")
-        client = self.create_client_token(endpoint, apikey, region)
+        client = self.create_client_token(endpoint, apikey, region, "https://api.microsofttranslator.com/")
```

Comment:
This change to the test would seem to indicate that the change in default could be breaking to some people?

Reply:
Test will pass even without the change. I have just added it here, to validate passing in the value. As it is only enabled test using audiences. (we have a work item to enable Entra ID tests as well in the future).